src/atmos_param/cloud_simple/cloud_simple.F90

```diff
@@ -203,7 +214,6 @@ subroutine calc_liq_frac(temp, frac_liq)
         frac_liq = 1.0 - (KELVIN-temp) / 40.0
```

Comment:
Perhaps add more spaces around the minus sign? Changing the `(KELVIN-temp)` to `(KELVIN - temp)`?